cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -342,6 +356,7 @@ public void run() {
                                                       }
                                                     }, cleanupJobSchedule, cleanupJobSchedule, TimeUnit.SECONDS
     );
+    authorizationEnforcementService.startAndWait();
```

Comment:
will any of the code above this do authorization enforcement? If so, better to move this as one of the first few statements.